### PR TITLE
Force pod update

### PR DESCRIPTION
**Pull request by calvix**
addressing https://github.com/giantswarm/giantswarm/issues/2453

requirements: update  tiller to 2.8.1 in every env ( I will take care)

### CODE REVIEW

**Comment by calvix** [comment]
This is fix, but we also need to install helm 2.8.1

File: service/installer/helm/helm.go [hunk 1]
```diff
@@ -269,11 +269,11 @@ func (i *HelmInstaller) Install(event eventerspec.DeploymentEvent) error {
 	// The arguments used to execute Helm for app installation can take multiple
 	// values files. At the end the command looks something like this.
 	//
-	//     helm upgrade --install --values ${file1} --values $(file2) ${project} ${chart_path}
+	//     helm upgrade --install --recreate-pods --values ${file1} --values $(file2) ${project} ${chart_path}
 	//
 	var installCommand []string
 	{
-		installCommand = append(installCommand, "upgrade", "--install")
+		installCommand = append(installCommand, "upgrade", "--install", "--recreate-pods")
```
[the remark this answers is not in the document]
Comment (calvix) [reply service/installer/helm/helm.go]: `--force` doesn't ensure that pod will be recreated it will ensure that resources will be updated that is in our case deployment

[the remark this answers is not in the document]
Comment (xh3b4sd) [reply service/installer/helm/helm.go]: I was under the impression that pods are recreated once the deployment is updated. It is then actually Kubernetes' responsibility to ensure this.

[the remark this answers is not in the document]
Comment (calvix) [reply service/installer/helm/helm.go]: If there is no real change in the deployment (same Sha of docker image) then there is no reason why k8s should roll the pod

[the remark this answers is not in the document]
Comment (calvix) [reply service/installer/helm/helm.go]: So its no rolling it. that is the whole root cause of the postmortem

File: helm/draughtsman-chart/templates/deployment.yaml [hunk 2]
```diff
@@ -45,6 +45,12 @@ spec:
             port: 8000
           initialDelaySeconds: 15
           timeoutSeconds: 1
+        lifecycle:
+          preStop:
+            exec:
+              command:
+              - sleep
+              - "5"
```
Comment (calvix) [inline helm/draughtsman-chart/templates/deployment.yaml]: This needs to be added because if we are updating draughtsman itself then pod will be killed before it can set the github deployment event to "successful" and this will create an infinite loop.

fix is  have proper handling of term signal in draughtsmen but that is out of scope fo this PR

Comment (calvix) [reply helm/draughtsman-chart/templates/deployment.yaml]: issue for tracking https://github.com/giantswarm/giantswarm/issues/2549

File: service/installer/helm/helm.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (xh3b4sd) [reply service/installer/helm/helm.go]: Good point. Ok cool. Thanks.

**Comment by calvix** [comment]
all tillers are upgrade and  this has been tested on all envs
